openshift/tests-extension/Makefile

```diff
@@ -42,7 +42,7 @@ TOOLS_BIN_DIR := $(CURDIR)/bin
 
 #SECTION Development
 .PHONY: verify #HELP To verify the code
-verify: tidy fmt vet lint
+verify: bindata verify-metadata tidy fmt vet lint
```

Comment:
To be useful in isolation, this needs to call `git diff` directly now that part of what it is doing is generating some data that we expect not to change.